swan-lake/development-tutorials/deployment-guide/serverless-deployment.md

```diff
@@ -0,0 +1,217 @@
+// returns a processed HTTP output to the caller.
+service / on new functions:HttpListener() {
+    resource function get hello(string name) returns string {
+        return string `Hello,  ${name}!`;
+    }
+}
+```
+3. Use `bal build` to generate the Azure Function artifacts.
+```
+$ bal build --cloud="azure_functions"
+```
+4. Execute the Azure CLI command given by the compiler to create and publish the functions by replacing `<function_app_name>` with your respective function app name.
+```
+$ func azure functionapp publish <function_app_name> --script-root target/azure_functions
+```
+5. Execute the command below to invoke the function by replacing <function_app_name> with your respective function app name.
+```
+$ curl https://<function_app_name>.azurewebsites.net/hello\?name\=Jack
+```
+
+### CI/CD with Azure Function
+
+This GitHub Action workflow automates the continuous integration and continuous deployment (CI/CD) process for a Ballerina project.
+It is triggered by every push to the repository and automatically builds the project.
+
+```yaml
+name: Ballerina CI/CD with Azure Functions
+
+on: [push]
+
+jobs:
+  build-and-deploy:
+    runs-on: ubuntu-latest
+
+    steps:
+      # Step 1: Checkout the Repository
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      # Step 2: Build the Ballerina Application
+      - name: Build Ballerina Application
+        run: bal build
+
+      # Step 3: Set up Azure Functions Core Tools
+      - name: Set up Azure Functions Core Tools
+        uses: philippelemaire/install-azure-functions-core-tools@v1
+
+      # Step 4: Deploy to Azure Functions
+      - name: Azure Functions Action
+        uses: Azure/functions-action@v1
+        with:
+          app-name: 'your-function-app-name'
+          package: './'
+        env:
+          AZURE_FUNCTIONAPP_PUBLISH_PROFILE: ${{ secrets.AZURE_FUNCTIONAPP_PUBLISH_PROFILE }}
+
+      # Optional: Test the Azure Function
+      - name: Invoke Azure Function
+        run: |
+          curl https://your-function-app-name.azurewebsites.net/api/your-function-name
+```
+- **Step 01:** The push event can initiate a build and deployment process
+- **Step 02:** This step compiles the Ballerina application to generate an executable JAR file. The `bal build` command compiles the code and packages it, making it ready for deployment.
+- **Step 03:** Install the Azure Functions Core Tools, which are necessary for deploying and testing Azure Functions in this GitHub Actions environment.
+- **Step 04:** Deploy the built Ballerina application to an Azure Function App. This step uploads the packaged function to Azure and configures it for serverless execution.
+
+For more reference,
+- See [AWS Lambda](https://ballerina.io/learn/aws-lambda/) for more information.
+- See [Azure Functions](https://ballerina.io/learn/azure-functions/) for information about deployment options that can be used in Azure Functions.
```

Comment:
Are we going to have above two pages in the learn section after these changes? If so what is the landing page location? And how do we handle the information loss (i.e. I see more information on those pages like supported triggers etc)